src/NLog.Web.AspNetCore/NLogRequestPostedBodyMiddleware.cs

```diff
@@ -59,8 +66,38 @@ public async Task Invoke(HttpContext context)
                 }
             }
 
-            // Execute the next class in the HTTP pipeline, this can be the next middleware or the actual handler
-            await _next(context).ConfigureAwait(false);
+            if (ShouldCaptureResponseBody(context))
```

Comment:
Leaning toward having this feature in its own Middleware. Maybe in its own NLog-extension. Because of the `RecyclableMemoryStream`-dependency.

Reply:
that is easy, it started in its own file, i can do that.